### PR TITLE
removed fold_change_type parameter, everything is log2 now

### PR DESCRIPTION
**Pull request by sean-mccorkle**
can we get this reviewed, and if passes, put into appdev beta for narrative checks?  Thanks!

### CODE REVIEW

File: lib/FeatureSetUtils/Utils/FeatureSetBuilder.py [hunk 1]
```diff
@@ -42,25 +42,22 @@ def _validate_upload_featureset_from_diff_expr_params(self, params):
...
+                  'p_cutoff', 'q_cutoff', 'fold_change_cutoff']:
             if p not in params:
                 raise ValueError('"{}" parameter is required, but missing'.format(p))
 
-        fold_scale_type = params.get('fold_scale_type')
-        validate_fold_scale_type = ['linear', 'logarithm']
-        if fold_scale_type not in validate_fold_scale_type:
-            error_msg = 'Input fold scale type value [{}] is not valid'.format(fold_scale_type)
-            raise ValueError(error_msg)
+        if params.get('fold_scale_type'):
```
Comment (scanon) [inline lib/FeatureSetUtils/Utils/FeatureSetBuilder.py]: Allow the parameter but require that it be log

Comment (sean-mccorkle) [reply lib/FeatureSetUtils/Utils/FeatureSetBuilder.py]: okay.  I'll put in the check there but not refer to it anywhere in the code at this point.

**Comment by sean-mccorkle** [comment]
Now allows the parameter as long as its "logarithm" -also added checks for this in the unit test script.

**Comment by scanon** [comment]
Good

**Pull request merged**